### DISCUSSION TITLE
DexcomOne and X-drip won't connect, Motorola e20, android 11

### DISCUSSION THREAD

**Discussion by brankaromic** [NightscoutFoundation/xDrip#2750] [Q&A]
![inbound6050443328287722396](https://user-images.githubusercontent.com/129010087/227807613-93055b6f-b935-4b9f-b7d4-aa2882b50156.jpg)
That problem is because xdrip persistently refuses to connect to the transmitter.
Connect once and take a reading and lose signal again. And so it goes round and round.
Sometimes it asks me to connect and pair and only when I manually click it recognize it and then shows it once and loses it again.
The cell phone is a Motorola e 20, android 11....

**Comment by Navid200** [comment]
Do you mind sharing your settings by posting a screenshot of Settings &#8722;> G5/G6/Dex1 Debug Settings?

**Reply by brankaromic** [reply]
![inbound8866051873287298698](https://user-images.githubusercontent.com/129010087/227944499-89b4b582-d317-4316-8695-e6d70658c32f.jpg)
![Uploading inbound7940255044983747641.jpg…]()

**Reply by brankaromic** [reply]
![Uploading inbound1969805775265138322.jpg…]()

**Reply by brankaromic** [reply]
![inbound5963769450145237443](https://user-images.githubusercontent.com/129010087/227945270-d075f526-0e82-4d22-af9a-64de843d75a7.jpg)

**Comment by brankaromic** [comment]
No I do not have.



pon, 27. ožu 2023. 00:46 Navid ***@***.***> je napisao:

**Reply by jamorham** [reply]
Please update to the latest nightly and confirm whether you have always had this problem or whether it just started recently. thanks

**Reply by brankaromic** [reply]
Hello.
Here I am with the feedback.
So like this, I installed the version from 23.3. It took a long time for him to connect to the transmitter, then he asked me to pair and connect and read the data once and the lost it again the next time.
Several times during the day it asked me to pair and connect and only when I clicked on that did the data come, and again one reading and then nothing again.
And so constantly.
It keeps asking for it as a manual pairing.And then again a missed opportunity. And so it goes round and round.
Then I installed version 28.3. last night.
And exactly the same thing.
Let's send photos for you to see.
Thank you.

**Reply by brankaromic** [reply]
![inbound2072899384686705691](https://user-images.githubusercontent.com/129010087/228541719-df5a210a-ca47-4dd7-90cd-332d869ad54c.jpg)
![inbound5547463099724711688](https://user-images.githubusercontent.com/129010087/228541720-96c15eed-63dc-4f4d-8c3a-9ea7b80d3767.jpg)
![inbound3395349722649858784](https://user-images.githubusercontent.com/129010087/228541757-3aeca954-adfa-4e20-9a3d-689af266a5ea.jpg)
![inbound2539359001169502398](https://user-images.githubusercontent.com/129010087/228541810-9c93bc97-b7a0-4be5-a3e0-bc381079b67d.jpg)
![inbound6927816207906637117](https://user-images.githubusercontent.com/129010087/228541916-c76fdc43-efdf-4373-826f-9ce4a639091c.jpg)
![inbound5084304013869547829](https://user-images.githubusercontent.com/129010087/228541946-82563053-7bc5-4638-a825-649a08329631.jpg)
![Uploading inbound1709883387380815842.jpg…]()

**Reply by brankaromic** [reply]
![Uploading inbound8566049203200978800.jpg…]()

**Reply by brankaromic** [reply]
![Uploading inbound1245293404761224254.jpg…]()

**Reply by brankaromic** [reply]
![inbound8057639375198028217](https://user-images.githubusercontent.com/129010087/228547196-e5ff4277-722b-4d07-86bd-76c4288762f8.jpg)

**Comment by nkmanolovsumup** [comment]
Hi, I have the exact same issue - started a Dexcom One sensor with the official app and now I'm attempting to connect to it via xdrip+ but to no avail. The `Bluetooth Link` status is `Disconnected` and remains so for hours. `Brain State: Deep Sleeping Errors...`. My goal is to use xdrip to calibrate the sensor. Any hints? I'm running the app on an OnePlus 6T phone with Android 11 and the default, recommended settings.

![Screenshot_20230625-194043](https://github.com/NightscoutFoundation/xDrip/assets/25202296/517b798c-d105-4819-b053-24e29e5037fb)

**Reply by Navid200** [reply]
But, why have you posted in an existing thread?  How is that going to help you or those who want to help you?
Please open a new thread, post a screenshot of the system status page, and I will provide everything I know.
